code/modules/events/vent_clog.dm

```diff
@@ -20,30 +17,23 @@
 	endWhen = rand(25, 100)
 	for(var/obj/machinery/atmospherics/components/unary/vent_scrubber/temp_vent in machines)
 		if(temp_vent.loc.z == ZLEVEL_STATION && !temp_vent.welded)
-			var/datum/pipeline/temp_vent_parent = temp_vent.PARENT1
-			if(temp_vent_parent.other_atmosmch.len > 20)
-				vents += temp_vent
+			vents += temp_vent
 	if(!vents.len)
 		return kill()
 
-/datum/round_event/vent_clog/tick()
-	if(activeFor % interval == 0)
-		var/obj/machinery/atmospherics/components/unary/vent = pick_n_take(vents)
-		while(vent && vent.welded)
-			vent = pick_n_take(vents)
-
+/datum/round_event/vent_clog/start()
+	for(var/obj/machinery/atmospherics/components/unary/vent in vents)
 		if(vent && vent.loc)
-			var/datum/reagents/R = new/datum/reagents(50)
+			var/datum/reagents/R = new/datum/reagents(1000)
 			R.my_atom = vent
-			R.add_reagent(pick(gunk), 50)
+			R.add_reagent(pick(get_random_reagent_id()), 1000)

```

Comment:
Keep the gunk list, and have a small chance of picking a different random reagent.

Otherwise, it'll just be harmless booze coming out of the vents.